### PR TITLE
add  and update the example for es6

### PR DESCRIPTION
**Pull request by geirman**
per #61 updated docs

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,40 +105,61 @@ public class MainApplication extends Application implements ReactApplication {
...
+ * @flow
+ */
+
+import React, { Component } from 'react';
+import { View, Button } from 'react-native';
+import Mailer from 'react-native-mail';
 
-var MailExampleApp = React.createClass({
-  handleHelp: function() {
+export default class App extends Component<{}> {
```
Comment (chirag04) [inline README.md]: also can we remove the type stuff. `<{}`. not many people are using them

File: README.md [hunk 2]
```diff
@@ -97,40 +105,61 @@ public class MainApplication extends Application implements ReactApplication {
...
       attachment: {
         path: '',  // The absolute path of the file from which to read data.
         type: '',   // Mime Type: jpg, png, doc, ppt, html, pdf
         name: '',   // Optional: Custom filename for attachment
       }
     }, (error, event) => {
-        if(error) {
-          AlertIOS.alert('Error', 'Could not send mail. Please send a mail to support@example.com');
-        }
+      console.log(error);
```
Comment (chirag04) [inline README.md]: remove?

**Comment by chirag04** [comment]
This is great! let's ship it after those nits.

**Comment by geirman** [comment]
sorry for the multiple commits. I missed your last suggestion, and realized that the example was missing the alert dependency. All should be good now though. Thanks for your work on this and for the quick merge :D

File: README.md [hunk 3]
```diff
@@ -97,40 +105,59 @@ public class MainApplication extends Application implements ReactApplication {
...
         type: '',   // Mime Type: jpg, png, doc, ppt, html, pdf
         name: '',   // Optional: Custom filename for attachment
       }
     }, (error, event) => {
-        if(error) {
-          AlertIOS.alert('Error', 'Could not send mail. Please send a mail to support@example.com');
-        }
+      Alert.alert(
+        'Error',
+        'Email could not be sent',
```
Comment (chirag04) [inline README.md]: @geirman sorry for bikeshedding this. wondering if we should simply show an alert with `error` and `event` in the message. Title as `Error` shows negative ideas. thoughts?

**Comment by geirman** [comment]
good idea, let me know if that's what you had in mind.

**Comment by chirag04** [comment]
thanks @geirman 👍

**Pull request merged**